### ISSUE TITLE
Tagging Versions

### ISSUE THREAD

**Issue by notmike101** [antomanc/bing-chat-sidebar-chrome-extension#14]
It might be a good idea to tag various versions using `git tag v#.#.#` (Following [semver guidelines](https://semver.org/)), potentially starting at `v2.0.0` if #11 is merged and `v1.1.0` if it is not. There is also the potential to going over previous commits and tagging where appropriate.

It would be helpful when tracking addition/removal of major or minor features. Would also let you track patch versions.

For example:
* The resolution to #11 would be increasing a major version since it is a big change that reworks a lot of the code.
* The resolution to #6 would be increasing a minor version since it reworks several APIs, but maintains the same functionality.
* The resolution to #5 would be increasing a patch version since it does not add new functionality or replace existing functionality.

Please discuss your opinions on this.

**Comment by antomanc** [comment]
Yes that would be a good idea, I'll add tags to those merge requests. I don't know about my previous commits because I was not following any particular rule and committing directly to the main branch. Thank you very much for all the support!
